guestbook/php-redis/Dockerfile

```diff
@@ -14,6 +14,9 @@
 
 FROM php:5-apache
 
+COPY fixup-apt-list.sh /
```

Comment:
I understand we don't have security updates for jessie release, but I'm wondering whether we can update to the latest version which has stretch as base image which does work without any of these crazy patches!

Reply:
the image can work well without these crazy patchers.
remove the update command.
test AMD64:
```
root@zkubetest:~# docker run gcr.io/google-samples/gb-frontend:v6
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
[Mon May 13 15:20:09.977625 2019] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.25 (Debian) PHP/5.6.40 configured -- resuming normal operations
[Mon May 13 15:20:09.977691 2019] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
^C[Mon May 13 15:20:12.871811 2019] [mpm_prefork:notice] [pid 1] AH00169: caught SIGTERM, shutting down
root@zkubetest:~#
```
test s390x:
```
root@d230718d690f:/home/liudali# docker run gcr.io/google-samples/gb-frontend-s390x:v6
AH00557: apache2: apr_sockaddr_info_get() failed for c3a219b07c49
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 127.0.0.1. Set the 'ServerName' directive globally to suppress this message
AH00557: apache2: apr_sockaddr_info_get() failed for c3a219b07c49
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 127.0.0.1. Set the 'ServerName' directive globally to suppress this message
[Mon May 13 15:15:24.111847 2019] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.10 (Debian) PHP/5.6.32 configured -- resuming normal operations
[Mon May 13 15:15:24.111879 2019] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
^C[Mon May 13 15:15:30.170413 2019] [mpm_prefork:notice] [pid 1] AH00169: caught SIGTERM, shutting down
root@d230718d690f:/home/liudali#
```
@mkumatag please review again.